packages/bot-skeleton/src/scratch/hooks/block_svg.js

```diff
@@ -67,10 +68,12 @@ Blockly.BlockSvg.prototype.showContextMenu_ = function (e) {
     // Save the current block in a variable for use in closures.
     const block = this;
     const menu_options = [];
+    if (!config.mainBlocks.includes(block.type) && this.isDeletable() && this.isMovable() && !block.isInFlyout) {
+        menu_options.push(Blockly.ContextMenu.blockDetachOption(block, e));
+    }
```

Comment:
i think we'll have to check with Anton where `Detach Block` option should be in the list. I dont think it will be that frequentely used to be the first

Reply:
And actually we can ask him if the current order is good and rise a card if it is not